doc/openapi/firecrest-developers-api.yaml

```
@@ -1530,7 +1530,33 @@ paths:
               description: Error
               schema:
                 type: string
-
+  '/storage/xfer-external/invalidate':
+    post:
+      summary: Invalidate temporary URL
+      description: >-
+        Remove a temporary URL attached to a given Task Id
+      tags:
+        - Storage
+      parameters:
+      - in: header
+        name: X-Task-Id
+        description: Task Id associated to the upload/download task
+        required: true
+        schema:
+          type: string
```

```suggestion
        - in: header
          name: X-Task-Id
          description: Task Id associated to the upload/download task
          required: true
          schema:
            type: string
```

Maybe add a level of padding for uniformity, since it's more indented in the other cases of this file.

---

added level of tab